appservice/src/createAppService/createAppService.ts

```diff
@@ -52,21 +64,31 @@ export async function createAppService(
                     learnMoreLink: 'https://aka.ms/Cfqnrc'
                 }
             ));
+            promptSteps.push(new AppServicePlanListStep());
+            promptSteps.push(new LocationListStep());
             break;
         case AppKind.app:
+            if (advancedCreation) {
+                promptSteps.push(new ResourceGroupListStep());
+                promptSteps.push(new SiteOSStep());
+                promptSteps.push(new SiteRuntimeStep());
+                promptSteps.push(new AppServicePlanListStep());
+                promptSteps.push(new LocationListStep());
+            } else {
+                if (workspace.workspaceFolders.length === 1) {
+                    // can make smart defaults if only one workspace is opened
+                    await setWizardContextDefaults(wizardContext);
+                }
+                promptSteps.push(new LocationListStep());
+                promptSteps.push(new SiteOSStep()); // will be skipped with a smart default
```

Comment:
I'm confused by this comment. Shouldn't it be "will be skipped _if there is_ a smart default"? I assume these steps are only included just in case you can't figure out the defaults...

Reply:
Correct